scifight/models.py

```diff
@@ -140,8 +146,6 @@ def __str__(self):
     class Meta:
         unique_together = ("room", "fight_num")
 
-    def __str__(self):
-        return "#{0}. {1}".format(self.fight_num, self.room)
```

Comment:
I insist on keeping this "__str__", 'cause overwise we get "Fight object" string displayed in comboboxes.